tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -577,117 +561,49 @@ def test_performance_issue_ids_array_with_undefined(self):
         response = self.do_request(query)
         assert response.status_code == 400, response.content
 
+    @override_options({"performance.issues.all.problem-creation": 1.0})
```

Comment:
I wonder if we should just override these features/options during test setup. I think we could do it here: https://github.com/getsentry/sentry/blob/253d97f4e447186a0f9194539bb02b0b225ebe06/src/sentry/utils/pytest/sentry.py#L28